image_load.py

```diff
@@ -0,0 +1,59 @@
+from pexelsapi.pexels import Pexels
+import requests
+import logging
+import random
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("Image_Load_Logger")
+
+
+class ImageException(Exception):
+    pass
+
+
+class APINotAvailable(ImageException):
+    pass
+
+
+class PexelAPIKeyError(ImageException):
+    pass
+
+
+def get_image_url(pexel_api_key):
+    try:
+        logger.info('Connecting to Pexels API...')
+        pexel = Pexels(pexel_api_key)
+        search_photos = pexel.curated_photos(page=1, per_page=80)
+        logger.info('Getting image url...')
+        image_url = search_photos['photos'][random.randint(0, 79)]['src']['tiny']

```

Comment:
On Pexels site they promise to add at least one new photo per hour to our curated list so that you always get a changing selection of trending photos.
But i figured out, that it,s not the first one always, so i take all the list and always take random image from it to reduse chance of it's repeating